LSP-eslint.sublime-settings

```diff
@@ -4,10 +4,18 @@
 			"languageId": "javascript",
 			"document_selector": "source.js",
 		},
+		{
+			"languageId": "javascriptreact",
+			"document_selector": "source.jsx | source.js.react",
+		},
 		{
 			"languageId": "typescript",
 			"document_selector": "source.ts",
 		},
+		{
+			"languageId": "typescriptreact",
+			"document_selector": "source.tsx",
+		},
```

Comment:
Shouldn't this also have `source.ts.react`? Or is there no such base scope out in the wild?

Reply:
I'm not sure but I've added it just in case. Also moved above `source.ts`: 70ccd4a7774b0a8609cb9ac175767d0b2937086b

Reply:
Hmm, looks like [this has been discussed back in May](https://github.com/sublimelsp/LSP-typescript/pull/17#discussion_r429550211) :smile: I'm using ST4's native TypeScript syntax right now, and the scope is `react.tsx`, don't think there's a need to support anything else, no?